authorization/repositories/repositories.go

```diff
@@ -2,15 +2,17 @@ package repositories
 
```

Comment:
Should be `package repository`. It's not recommended to use plurals in package names in go.